specs/resources/aliases/list.json

```diff
@@ -1,6 +1,6 @@
 {
     "get": {
-        "tags": [ "entities" ],
+        "tags": [ "aliases" ],
```

Comment:
This reminds me of the discussion we had in https://github.com/wmde/wikibase-rest-api-proposal/pull/74#pullrequestreview-493424722 ff. except that this time it is not an "internal" matter but users get presented with e.g. "aliases" as apparent equals to "items". Of course, the path is visible but, at least in my way of interpreting what it displays, it makes it harder to approach the data model instead of allowing the user to slowly dig into.

I created #79 (which has its own challenges) to illustrate how I - roughly - thought things would turn out.

Reply:
First of all sorry for the wall of text, I really want to try to address this topic from as many perspectives as possible, and also give a review to your suggestion PR, so it turned out a bit long.

1. The suggested restructure is meant to make the current UI more digestible to readers of this proposal, as it  (I can almost fully guarantee) is going to be assessed through the deployed UI rather than by reading the specs.

2. According to the [OpenAPI specifications](https://swagger.io/specification/#operation-object), tags are most and foremost an organizational aid for the parsing tools: 

    > "A list of tags for API documentation control. Tags can be used for logical grouping of operations by resources or any other qualifier."

    And in the [root object](https://swagger.io/specification/#openapi-object): 

    > "A list of tags used by the specification with additional metadata. The order of the tags can be used to reflect on their order by the parsing tools."

3. As much as I agree with you that the overriding categorization of the endpoints should be made clearer, I think that in this case in your suggestion PR, which is really great at identifying the main resources of the datamodel, you are trying to force something into the UI tool that it doesn't currently support, namely, hierarchical information structuring or, simply put, categories. 

    Therefore, the issue is with the presentation of the information rather than the tags themselves. And unfortunately, I think that tagging resources by their parent only increases the problem that this PR is meant to solve (large chunks of endpoints that deal with semantically distinguishable resources) and makes the specification harder to read due to unnecessary duplication.

    In order to solve this - agreeably annoying - shortcoming, I think we will simply have to find a better tool to parse and render this information and allow categorization, which Is why I suggested the investigation into separate "pages" in the first place. But for now, as it is currently out of scope, I think we can agree on giving the readers of the specification some credit in regards to understanding that some resources are in fact "sub-resources".

4. Lastly, as mentioned before, we are listing labels / aliases / descriptions / etc. as a resource that can be fully interacted with, and the hierarchy between them and items / properties / statements is fully expressed in the path (and somewhat also by the order of presentation enforced by tag ordering in this PR). The need to identify one before being able to access the other also contributes to the clarity of these relationships.

    However, and this is maybe the bigger question to ask here, if we don't see these as full resources, why did we include endpoints to access and manipulate them in the first place?

Reply:
I would agree with both

> Therefore, the issue is with the presentation of the information rather than the tags themselves

and 

> tags are most and foremost an organizational aid for the parsing tools

Consequently the discrepancy is really not in the technology but in the output we strive for (how we cut the "categories"). As this is largely a matter of taste (and maybe philosophy) I'm happy to go with your approach instead of trying to sell mine.